composer.json

```diff
@@ -55,7 +56,8 @@
             "common\\http\\": "common/http",
             "entities\\": "entities",
             "services\\": "services",
-            "repositories\\": "repositories"
+            "repositories\\": "repositories",
+            "OpenEMR\\Encounter\\Services\\": "interface/patient_file/encounter/Services"
```

Comment:
I'm not happy with this location either, but I wanted to keep the encounter stuff together for the time being